p4-16/spec/P4-16-draft-spec.mdk

```diff
@@ -3085,8 +3085,8 @@ tuple<bit<32>, bool> x = { 10, false };
 
 ## Operations on sets { #sec-set-exprs }
 
-Some P4 expressions denote sets of values (```set<T>```, for some type
-```T```; see Section [#sec-synth-types].1). These expressions can
+Some P4 expressions denote sets of values (```set<T>```, for some type ```T```;
+see Section [#sec-synth-types].1). These expressions can
```

Comment:
It worked fine.  One less manual cross-ref in the document.  I noticed you had been fixing those as you found them.